common/constants/index.js

```diff
@@ -1,3 +1,5 @@
 const headerHeight = '88px'
 
-export { headerHeight }
+const API_URL = `https://app-co-api.herokuapp.com/api`
```

Comment:
I would much prefer to use the `apiServer` provided in Redux - this makes local development much smoother.

Reply:
oh yeah sure thing

Reply:
This has been resolved